packages/shared_preferences/shared_preferences_android/lib/src/strings.dart

```diff
@@ -0,0 +1,12 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// String prefix for lists that are encoded with json in dart.
+const String jsonListPrefix = 'VGhpcyBpcyB0aGUgcHJlZml4IGZvciBhIGxpc3Qu!';
```

Comment:
This is a 3rd place where these strings are found. If they are not being generated from a file someplace consider linking to each location from every other location in a comment.